packages/@lwc/engine-server/src/__tests__/fixtures/getter-is-connected/expected.html

```diff
@@ -1,3 +1,7 @@
 <x-getter-is-connected>
-  <template shadowroot="open">Is connected? true</template>
+  <template shadowroot="open">
+    <ul>
+      <li>connectedCallback</li>
```

Comment:
IMO, the test should directly test that isConnected property
- in constructor is false
- in connectedCallback is true